xla/stream_executor/dnn.h

```diff
@@ -1616,7 +1616,8 @@ class DnnSupport {
       const dnn::BatchDescriptor& output_descriptor,
       DeviceMemoryBase output_data,
       const dnn::ConvolutionDescriptor& convolution_descriptor,
-      bool use_fallback, ScratchAllocator* scratch_allocator,
+      bool use_fallback, 
```

Comment:
nit: please undo the changes here, they seem to be from the previous version

Reply:
Done